serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+
+</Message>
+
+You can recreate your credentials either via the [console](https://console.scaleway.com/) (once the new version is in production) or via the new `v1beta1` API directly. In terms of the API, the [current `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version will continue to be available 30 days after migration/General Availability, co-existing with the new `v1beta1` version. The documentation for both will be accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api) via a dropdown at the top of the Messaging and Queuing API page. 
+
+Note the following important information about the migration/crossover period, when both versions of the API will exist:
+
+**Before release of new version**
+- Only the `v1alpha1` API is available. All your resources exist in this API.
+- The console is based on this API, so when you view your resources in the console, you are viewing resources in the `v1alpha1` API.
+
+**At the time that SQS and NATS go into GA**
+- The `v1alpha1` API is still available. Your resources continue to exist in this API until you delete them.
+- The `v1beta1` API also becomes available. You should recreate your resources in this API.
+- The console now uses the `v1beta1` API. **You will no longer be able to see or manage your old resources from the `v1alpha1` API via the console**. Any resources now created via the console exist in the `v1beta1` API.
+
+**30 days afterwards**
+- The `v1alpha1` API is decommissioned. Any resources still existing on this API are deleted.
+
+**The URL/endpoints for your Messaging and Queuing SNS service and for your topics will not be the same in the two versions**. Make sure that when you recreate your resources with the new version, you remember to update the endpoint in any code, scripts, applications and architectures that were using  the old endpoint. If you do not do so, they will continue to access the "old" resources rather than the new ones you have created for the new version.
+
+The new URLs/endpoints will be visible in the console and also from the new API.
+
+**Summary: SNS**
+
+✅ You must migrate from the old version to the new by recreating your resources and credentials <br />
+✅ You must update your code, applications and architectures so they use the endpoints of the new version once you migrate <br />
+✅ SNS will remain in Public Beta (in both old and new versions) and will not yet be billed <br />
+✅ All resources still existing on the old version will be deleted 30 days after the beginning of General Availability
+
+## Terraform
+
+There will be a new Terraform provider for the new version of Messaging and Queuing. The link will be published shortly.
+
+If you manage Messaging and Queuing using the old Terraform provider, you will need to recode your Terraform files to update to the new provider, in line with the migration timelines laid out above.
```

Comment:
Is `recode` correct? (Just asking cause my grammar reviewer flagged this, and I couldn't find matching results on google; only `record` or `convert`).

Reply:
Hmm to me it sounds kind of OK like "code again" - I found a few google results but yes nothing in the dictionary. I'll look for an alternative.

Reply:
@RoRoJ no need to replace it, then. I flagged it just in case. Thank you!